### PR TITLE
Multitest helpers

### PR DESCRIPTION
**Pull request by jawoznia**

### CODE REVIEW

File: contracts/cw1/Cargo.toml [hunk 1]
```diff
@@ -12,4 +12,8 @@ homepage = "https://cosmwasm.com"
 cosmwasm-std = { version = "1.1", features = ["staking"] }
 cosmwasm-schema = "1.1"
 serde = { version = "1", default-features = false, features = ["derive"] }
-sylvia = { version = "0.2.2", path = "../../sylvia" }
+sylvia = { version = "0.2.2", path = "../../sylvia", features = ["mt"] }
```
Comment (hashedone) [inline contracts/cw1/Cargo.toml:15]: We don't want Sylvia to have "mt" in a normal build. Add it to dev-dependencies, and add an additional feature "mt" feature which transitively enables "mt" feature for Sylvia.

**Review by hashedone — COMMENTED** [review]
Seems mostly ok, but I would like to see the usage of emitted stuff here, so we see it works as we want it to work. Also important comment about deps.

File: contracts/cw1/Cargo.toml (hunk 1 above)
Comment (jawoznia) [reply contracts/cw1/Cargo.toml]: As for example this is only part of implementation containing helpers for traits.
There will be a second delivery with the multitest helpers generation for contracts. Then there will be examples.

**Comment by jawoznia** [comment]
part of #27

**Review by hashedone — APPROVED** [review]
For now it is fine, but in follow up we need examples of usage of that (but I get we need contract stuff generation for that)

**Pull request merged**